case-server/src/main/java/com/powsybl/caseserver/CaseInfos.java

```diff
@@ -20,7 +24,18 @@
 @ApiModel("Case infos")
 public class CaseInfos {
 
-    private String name;
+    private static final String NAME_HEADER_KEY     = "name";
+    private static final String FORMAT_HEADER_KEY   = "format";
 
+    private String name;
     private String format;
+
+    @Transient
+    public Message<String> getMessage() {
```

Comment:
I would not add this method as an instance method but a static one, because it forces you to add a @Transient

Reply:
This method must be an instance method because it uses  **name and format** attributes

Reply:
public Message<String> getMessage(CaseInfos caseInfos)
You have access to name and format through getters

Reply:
To be more precise, for me when you add @Transient to a bean it often means you have a design issue.

Reply:
Done